documentation/book/proc-uninstalling.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is included in the following assemblies:
+//
+// master.adoc
+
+[id='ininstalling-{context}']
+= Uninstalling {ProductName}
+
+. Identify resources:
++
+--
+* Referenced by `Kafka`, `KafkaConnect` or `KafkaMirrorMaker`
+* Consumed by the operators
+--
++
+These include resources specific to a deployment, such as:
++
+* Secrets (Custom CAs and certificates, Kafka Connect secrets, and other secrets)
+* Logging `ConfigMaps`
+
+. Delete the cluster operator `Deployment`, related `CustomResourceDefinitions` and `RBAC` resources:
++
+[options="nowrap",subs="+quotes,attributes"]
+----
+{cmdcli} delete -f install/cluster-operator
+----
++
+Deleting `CustomResourceDefinitions` results in the garbage collection of the corresponding custom resources (`Kafka`, `KafkaConnect` or `KafkaMirrorMaker`) and the resources dependent on them (Deployments, StatefulSets, and other dependent resources).
```

Comment:
This should be probably marked as warning or important. The list of resources should also list `KafkaConnectS2I`.

Reply:
Updated